login-workflow/package.json

```diff
@@ -56,28 +56,28 @@
     "doc": "docs"
   },
   "peerDependencies": {
-    "@brightlayer-ui/angular-components": ">=8.0.0",
+    "@brightlayer-ui/angular-components": "9.0.0",
```

Comment:
This is an exact version...it should probably be ^9.0.0

Reply:
good catch